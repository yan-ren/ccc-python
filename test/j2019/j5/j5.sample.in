AA AB
AB BB
B AA
4 AB AAAB
