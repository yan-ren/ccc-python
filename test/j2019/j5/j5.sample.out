2 1 BB
3 2 BAA
3 1 AAAA
1 3 AAAB
